pinot-connectors/pinot-spark-connector/src/main/scala/org/apache/pinot/connector/spark/connector/PinotServerDataFetcher.scala

```diff
@@ -46,7 +47,9 @@ private[pinot] class PinotServerDataFetcher(
   private val brokerId = "apache_spark"
   private val metricsRegistry = PinotMetricUtils.getPinotMetricsRegistry
   private val brokerMetrics = new BrokerMetrics(metricsRegistry)
-  private val queryRouter = new QueryRouter(brokerId, brokerMetrics)
+  private val pinotConfig = new PinotConfiguration()
+  private val serverRoutingStatsManager = new ServerRoutingStatsManager(pinotConfig)
```

Comment:
Can we leave null as opposed to initializing it here / changing this file ?

Reply:
If we pass null, we will have to perform a if/else null check everywhere serverRoutingStatsManager is used. Leaving as such based on offline discussion.